### PR TITLE
Fixing the VideoViewer activity's event bubbling issue

### PR DESCRIPTION
**Pull request by ricknjacky**
On touch screen, when you play a video then touch the close button to go back to the main screen, the fullscreen mode is no longer activated.
Tested on touch screen.

issue #859 solved :)

### CODE REVIEW

**Comment by ricknjacky** [comment]
b25e336 is the fix here for the issue #859

File: activities/SharedNotes.activity/locale.ini [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [*]
-NotesSaved=Image was saved to journal!
+NotesSaved = Your Notes image was exported
```
Comment (llaske) [inline activities/SharedNotes.activity/locale.ini:2]: This update is not related to this fix.

File: activities/SharedNotes.activity/locale.ini [hunk 2]
```diff
@@ -46,7 +46,7 @@ PlayerJoin={{user}} joined
 PlayerLeave={{user}} left
 
 [en]
-NotesSaved=Image was saved to journal!
+NotesSaved = Your Notes image was exported
```
Comment (llaske) [inline activities/SharedNotes.activity/locale.ini:49]: This update is not related to this fix.

**Review by llaske — CHANGES_REQUESTED** [review]
Good job. It seems to work.
BTW your branch is not up to date with the `dev` one so there is an unrelated update in your PR.

File: activities/SharedNotes.activity/locale.ini (hunk 2 above)
Comment (ricknjacky) [reply activities/SharedNotes.activity/locale.ini:49]: I know its a conflict I realised later after the push sorry

**Comment by ricknjacky** [comment]
I realise, sorry

**Comment by ricknjacky** [comment]
@llaske should I push a new PR?

**Comment by llaske** [comment]
You could update your branch or you could just update unrelated files to go back to their initial version.
